### PR TITLE
add Scenario Description

### PR DESCRIPTION
**Pull request by pepezzzz**
### What is changed, added or deleted? (Required)

add scenario description

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: zh/usage-scenario-shard-merge.md [hunk 1]
```diff
@@ -37,14 +37,14 @@ aliases: ['/docs-cn/tidb-data-migration/dev/usage-scenario-shard-merge/']
...
-1. 合并三个实例中的 `user`.`information` 表至下游 TiDB 中的 `user`.`information` 表。
-2. 合并三个实例中的 `user`.`log_{north|south|east}` 表至下游TiDB中的 `user`.`log_{north|south|east}` 表。
-3. 合并三个实例中的 `store_{01|02}`.`sale_{01|02}` 表至下游TiDB中的 `store`.`sale` 表。
-4. 过滤掉三个实例的 `user`.`log_{north|south|east}` 表的所有删除操作。
-5. 过滤掉三个实例的 `user`.`information` 表的所有删除操作。
-6. 过滤掉三个实例的 `store_{01|02}`.`sale_{01|02}` 表的所有删除操作。
-7. 过滤掉三个实例的 `user`.`log_bak` 表。
-8. 因为 `store_{01|02}`.`sale_{01|02}` 表带有 bigint 型的自增主键，将其合并至 TiDB 时会引发冲突。你需要有相应的方案来避免冲突。
+1. 同名表合并场景，三个实例中的 `user`.`information` 表合并至下游 TiDB 中的 `user`.`information` 表。
+2. 不同名表合并场景，三个实例中的 `user`.`log_{north|south|east}` 表合并至下游TiDB中的 `user`.`log_{north|south|east}` 表。
```
Comment (yikeke) [inline zh/usage-scenario-shard-merge.md:41]:
```suggestion
2. 不同名表合并场景，比如将三个实例中的 `user`.`log_{north|south|east}` 表合并至下游 TiDB 中的 `user`.`log_{north|south|east}` 表。
```

File: zh/usage-scenario-shard-merge.md [hunk 2]
```diff
@@ -37,14 +37,14 @@ aliases: ['/docs-cn/tidb-data-migration/dev/usage-scenario-shard-merge/']
...
+3. 分片表合并场景，三个实例中的 `store_{01|02}`.`sale_{01|02}` 表合并至下游TiDB中的 `store`.`sale` 表。
```
Comment (yikeke) [inline zh/usage-scenario-shard-merge.md:42]:
```suggestion
3. 分片表合并场景，比如将三个实例中的 `store_{01|02}`.`sale_{01|02}` 表合并至下游 TiDB 中的 `store`.`sale` 表。
```

File: zh/usage-scenario-shard-merge.md [hunk 3]
```diff
@@ -37,14 +37,14 @@ aliases: ['/docs-cn/tidb-data-migration/dev/usage-scenario-shard-merge/']
...
+4. 删除操作过滤场景，过滤掉三个实例的 `user`.`log_{north|south|east}` 表的所有删除操作。
+5. 删除操作过滤场景，过滤掉三个实例的 `user`.`information` 表的所有删除操作。
+6. 删除操作过滤场景，过滤掉三个实例的 `store_{01|02}`.`sale_{01|02}` 表的所有删除操作。
```
Comment (yikeke) [inline zh/usage-scenario-shard-merge.md:45]:
```suggestion
4. 过滤删除操作场景，比如过滤掉三个实例中 `user`.`log_{north|south|east}` 表的所有删除操作。
5. 过滤删除操作场景，比如过滤掉三个实例中 `user`.`information` 表的所有删除操作。
6. 过滤删除操作场景，比如过滤掉三个实例中 `store_{01|02}`.`sale_{01|02}` 表的所有删除操作。
```

File: zh/usage-scenario-shard-merge.md [hunk 4]
```diff
@@ -37,14 +37,14 @@ aliases: ['/docs-cn/tidb-data-migration/dev/usage-scenario-shard-merge/']
...
+7. 表名通配符选中后的过滤场景，`user`.`log_*` 过滤掉三个实例的 `user`.`log_bak` 表。
```
Comment (yikeke) [inline zh/usage-scenario-shard-merge.md:46]:
```suggestion
7. 使用通配符过滤特定表的场景，比如使用通配符 `user`.`log_*` 过滤掉三个实例的 `user`.`log_bak` 表。
```

File: zh/usage-scenario-shard-merge.md [hunk 5]
```diff
@@ -37,14 +37,14 @@ aliases: ['/docs-cn/tidb-data-migration/dev/usage-scenario-shard-merge/']
...
+8. 主键冲突处理场景，假设 `store_{01|02}`.`sale_{01|02}` 表带有 bigint 型的自增主键，将其合并至 TiDB 时会引发冲突，使用相应的方案来避免冲突。
```
Comment (yikeke) [inline zh/usage-scenario-shard-merge.md:47]:
```suggestion
8. 主键冲突处理场景，假设 `store_{01|02}`.`sale_{01|02}` 表带有 bigint 型的自增主键，将其合并至 TiDB 时会引发冲突，可以使用相应的方案来避免冲突。
```

File: zh/usage-scenario-shard-merge.md [hunk 6]
```diff
@@ -37,14 +37,14 @@ aliases: ['/docs-cn/tidb-data-migration/dev/usage-scenario-shard-merge/']
...
 
-1. 合并三个实例中的 `user`.`information` 表至下游 TiDB 中的 `user`.`information` 表。
-2. 合并三个实例中的 `user`.`log_{north|south|east}` 表至下游TiDB中的 `user`.`log_{north|south|east}` 表。
-3. 合并三个实例中的 `store_{01|02}`.`sale_{01|02}` 表至下游TiDB中的 `store`.`sale` 表。
-4. 过滤掉三个实例的 `user`.`log_{north|south|east}` 表的所有删除操作。
-5. 过滤掉三个实例的 `user`.`information` 表的所有删除操作。
-6. 过滤掉三个实例的 `store_{01|02}`.`sale_{01|02}` 表的所有删除操作。
-7. 过滤掉三个实例的 `user`.`log_bak` 表。
-8. 因为 `store_{01|02}`.`sale_{01|02}` 表带有 bigint 型的自增主键，将其合并至 TiDB 时会引发冲突。你需要有相应的方案来避免冲突。
+1. 同名表合并场景，三个实例中的 `user`.`information` 表合并至下游 TiDB 中的 `user`.`information` 表。
```
Comment (yikeke) [inline zh/usage-scenario-shard-merge.md:40]:
```suggestion
1. 同名表合并场景，比如将三个实例中的 `user`.`information` 表合并至下游 TiDB 中的 `user`.`information` 表。
```

**Comment by ireneontheway** [comment]
@pepezzzz Hi. Is this PR related to DM v2.0? Or just a daily upgrade?

**Comment by csuzhangxc** [comment]
> @pepezzzz Hi. Is this PR related to DM v2.0? Or just a daily upgrade?

I think it's just a daily upgrade.

**Pull request merged**